test/api_test.py

```diff
@@ -79,3 +81,70 @@ def test_peer_remove(gd2client):
     for node in node_list:
         _, resp = gd2client.peer_remove(peerid=Peer.get_peer_id(node))
         assert not bool(resp)
+
+
+def test_georep_create_delete(gd2client):
+    """Test for georep session operations."""
+    m_bricks = []
+    _, resp = gd2client.peer_status()
+    peer_id = resp[0]["id"]
+    for i in range(3):
+        temp_path = tempfile.mkdtemp(prefix=str(i), dir="/tmp/")
```

Comment:
why are you creating the directories in local machine?

Reply:
I wasn't aware of the fact that gd2 will be handling the creation of directories, in that case, what is expected to be sent against the bricks field? the default values will be used?